### PR TITLE
Gui 1472 Magic search, facet updates and landing page header re-org

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-1472

### CODE REVIEW

**Comment by dkavanagh** [comment]
Looks like CI got another "waitforvisible" failure. Should be good to merge.. can't we fix the test?

File: eucaconsole/forms/volumes.py [hunk 1]
```diff
@@ -226,8 +225,7 @@ def get_status_choices():
         return [
             {'key':'creating', 'label':'Creating'},
             {'key':'available', 'label':'Available'},
-            {'key':'in-use', 'label':'In use'},
-            {'key':'deleting', 'label':'Deleting'},
-            {'key':'deleted', 'label':'Deleted'},
-            {'key':'error', 'label':'Error'},
+            {'key':'attached', 'label':'Attached'},
```
Comment (kamalgill) [inline eucaconsole/forms/volumes.py]: @dkavanagh: The labels should be marked for i18n

File: eucaconsole/forms/stacks.py [hunk 2]
```diff
@@ -71,5 +71,30 @@ class StacksFiltersForm(BaseSecureForm):
     def __init__(self, request, **kwargs):
         super(StacksFiltersForm, self).__init__(request, **kwargs)
         self.facets = [
-            {'name': 'tags', 'label': self.tags.label.text}
+            {'name': 'stack_status', 'label': _(u'Status'), 'options':[
+                {'key':'CREATE_IN_PROGRESS', 'label':'Create in progress'},
```
Comment (kamalgill) [inline eucaconsole/forms/stacks.py]: @dkavanagh: The labels should be marked for i18n.  And while we're here, let's remove the extra line breaks below.

File: eucaconsole/forms/images.py [hunk 3]
```diff
@@ -91,7 +91,6 @@ def __init__(self, request, cloud_type='euca', **kwargs):
             {'name':'platform', 'label':self.platform.label.text, 'options':self.get_platform_choices()},
             {'name':'architecture', 'label':self.architecture.label.text, 'options':self.get_architecture_choices()},
             {'name':'root_device_type', 'label':self.root_device_type.label.text, 'options':self.get_root_device_type_choices()},
```
Comment (kamalgill) [inline eucaconsole/forms/images.py]: @dkavanagh: This block has numerous PEP8 violations (e.g. no space between key/value in dict items, line length > 120 chars)

File: eucaconsole/forms/instances.py [hunk 4]
```diff
@@ -393,21 +393,23 @@ def __init__(self, request, ec2_conn=None, autoscale_conn=None,
...
-                'options': self.getOptionsFromChoices(vpc_choices)},
-        )
         if cloud_type=='euca':
             self.facets.append(
                 {'name':'roles', 'label':self.roles.label.text,
                     'options':self.getOptionsFromChoices(self.iam_choices_manager.roles(add_blank=False))},
             )
+        if BaseView.is_vpc_supported(request):
+            self.facets.append(
+                {'name':'subnet_id', 'label':self.subnet_id.label.text,
```
Comment (kamalgill) [inline eucaconsole/forms/instances.py]: @dkavanagh: Please add a space between dict key/value pairs here and elsewhere for PEP8 compliance.  I wonder if the flake8 filter you're using ignores those violations?

**Comment by kamalgill** [comment]
@dkavanagh: Good to merge after conflicts are resolved

**Pull request merged**